sszgen/main.go

```diff
@@ -388,6 +396,9 @@ func (e *env) print(first bool, order []string, experimental bool) (string, bool
 
 	// Print the objects in the order in which they appear on the file.
 	for _, name := range order {
+		if exclude := e.excludeTypeNames[name]; exclude {
```

Comment:
I wonder if logs could be helpful during the gen process. Maybe not in this PR, but in methodical we could use some logging to explain certain things happening throughout the lifecycle of a parse/generate/render call. Here we could log "excluding type X found in file"

Reply:
sounds good to me!